src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -241,6 +243,7 @@
         "fileChanges": {
           "description": "A message relevant to this annotation.",
           "type": "array",
+          "uniqueItems": true,
```

Comment:
No, it's the `replacement` objects that must be in an ordered list, not the `fileChange` objects.